README.md

```diff
@@ -42,81 +80,115 @@ Before using the command you have to substitute words in uppercase with proper o
 
 The value of the `refresh_token` field in the response is the offline token.
 
-## Usage
+### Email configuration
 
-First make sure to copy luigi.cfg-sample and update it.
+Email configuration can be created by coping the `email_config.cfg-sample`:
 
 ``` bash
-$ cp luigi.cfg-sample luigi.cfg
+$ cp email_config.cfg-sample email_config.cfg
+```
+
+Config options overview:
+
+
+| Variable | Section | Default value               | Description                                                   |
+|----------|---------|-----------------------------|---------------------------------------------------------------|
+| log_file | global  | python.log                  | Logging file name.                                            |
+| receiver | email   |                             | Email address of the receiver, can be a comma separated list. |
+| sender   | email   |                             | Email address of the sender.                                  |
+| prefix   | email   | [CSR Data Loading Pipeline] | Prefix for subject line of the error email.                   |
+| port     | smtp    | 587                         | Port to use for sending emails.                               |
+| username | smtp    |                             | Username for email client, when not needed can be left empty. |
+| password | smtp    |                             | Password for email client.                                    |
+| host     | smtp    | smtp.gmail.com              | Host of the email client.                                     |
+
+
+## Input data
+
+In the drop directory (`drop_dir` in luigi.cfg) the clinical data files should be provided as well as a folder called NGS with all omcs data. 
+Each file in the drop directory has to be accompanied by a sha1 checksum file.
+
+File naming convention:
+- File name: `<filename>.<extension>`
+- Sha1 file: `<filename>.<extension>.sha1`
+
+E.g. data.txt has to have data.txt.sha1 next to it with sha1 hash of the data file. 
+As sha1 hashes 40 characters long the rest of the file gets ignored:
+
+`1625be750dab24057c4c82d62d27298236ebb04c diagnosis.txt`
 
+For more information, see the [CSR data model](https://github.com/thehyve/python_csr2transmart#data-model) description 
+and an example of [input data files](./test_data/dropzone/E2E_TEST_DATA).
+
+## Usage
+
+1. Make sure you have `luigi.cfg` and `email_config.cfg` properly configured (see [configuration section](#configuration)) 
+and the input data is in the proper directory (see [input data section](#input-data)).
+
+2. Install dependencies:
+
+Pipeline requires Python >= 3.6.
+
+``` bash
+$ python -m pip install -r requirements/requirements.txt
```

Comment:
The `$` symbol makes it look as a shell line, but also makes it hard to copy-paste the command.